backend/api/src/handlers/dynamodb/put-service-client.ts

```diff
@@ -34,8 +34,8 @@ export type putServiceHandlerInvokeEvent = {
     body: string;
 };
 
-export const putServiceClientHandler = async (event: putServiceHandlerInvokeEvent): Promise<APIGatewayProxyResult> => {
-    const payload: clientRegistryRegistrationResponse = JSON.parse(event.body);
+export const putServiceClientHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
```

Comment:
I added a different type on `event` previously as this lambda was not invoked by API gateway but instead invoked by a step function with a defined JSON Payload, is there a reason this has changed back?

Reply:
Thanks. Yes: this makes sense and I've worked through cleaning these up.

Reply:
Thanks. Yes: this makes sense and I've worked through cleaning these up.